.github/workflows/node.js.yml

```diff
@@ -84,6 +84,12 @@ jobs:
                   git remote add forkUpstream https://github.com/$REPO_NAME  # URL of the fork
                   git fetch forkUpstream  # Fetch fork
 
+            - name: Merge in target branch to avoid false negatives.
+              env:
+                  TARGET_BRANCH: ${{ github.event.pull_request.base.ref }}
+              run: |
```

Comment:
```suggestion
              # Note: "git merge" should always succeed here, because GHA won't
              # start the job if there are merge conflicts. https://github.com/orgs/community/discussions/11265
              run: |
```